### PR TITLE
[5.6] Update php doc for Container/Container.php setInstance

### PR DESCRIPTION
**Pull request by TBlindaruk**

### CODE REVIEW

File: src/Illuminate/Container/Container.php [hunk 1]
```diff
@@ -1181,7 +1181,7 @@ public static function getInstance()
      * Set the shared instance of the container.
      *
      * @param  \Illuminate\Contracts\Container\Container|null  $container
-     * @return static
+     * @return static|ContainerContract
```
Comment (a-komarev) [inline src/Illuminate/Container/Container.php]: Shouldn't it be fully qualified class name here?

Comment (TBlindaruk) [reply src/Illuminate/Container/Container.php]: @a-komarev , I`m not sure. Because ContainerContract already used in this file.

Comment (a-komarev) [reply src/Illuminate/Container/Container.php]: Yes, but it's an alias, not a FQCN, but maybe it's just my vision of the code styling.

Comment (mputkowski) [reply src/Illuminate/Container/Container.php]: It should be FQCN like in method's parameter.

Comment (TBlindaruk) [reply src/Illuminate/Container/Container.php]: In method it is a 
```PHP
ContainerContract $container = null
```

Should I really change it to the 
```PHP
Illuminate\Contracts\Container\Container
```
in Doc?

Comment (TBlindaruk) [reply src/Illuminate/Container/Container.php]: Changed

**Pull request merged**